packages/account/src/Components/formik-confirmation-checkbox/formik-confirmation-checkbox.tsx

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+import { Checkbox, Text } from '@deriv/components';
+import { isEmptyObject, isMobile } from '@deriv/shared';
+import { useFormikContext } from 'formik';
+
+/**
+ * Props for the confirmation checkbox component.
+ */
+type TConfirmationCheckboxProps = {
+    /**
+     * A boolean value indicating whether the checkbox is confirmed or not.
+     */
+    confirmed: boolean;
+    /**
+     * A React.SetStateAction function to set the confirmed state of the checkbox.
+     */
+    setConfirmed: React.Dispatch<React.SetStateAction<boolean>>;
+    /**
+     * The label of the checkbox.
+     */
+    label: string;
+};
+
+/**
```

Comment:
@yashim-deriv @ali-hosseini-deriv @farzin-deriv any suggestion if we can keep this kind of docs or anything can be improved or changed

Reply:
if this is ok, we should encourage all devs to make such things in other packages as well :) 
just my opinion

Reply:
Same here, Although this component is more generic but it exists in the account package, Maybe you can move it to the components package and then use storybook instead of commenting in the code. 

In my opinion, TS + TSDoc at type level is enough for most of the cases, You can ignore this comment as well 🙇🏻

Reply:
Thanks @farzin-deriv for your input.